src/SASViyaApiClient.ts

```diff
@@ -1238,10 +1249,17 @@ export class SASViyaApiClient {
   private async pollJobState(
     postedJob: any,
     etag: string | null,
-    accessToken?: string
+    accessToken?: string,
+    pollOptions?: PollOptions
   ) {
-    const MAX_POLL_COUNT = 1000
-    const POLL_INTERVAL = 100
+    let MAX_POLL_COUNT = 1000
+    let POLL_INTERVAL = 100
+
+    if (pollOptions) {
+      MAX_POLL_COUNT = pollOptions.MAX_POLL_COUNT
```

Comment:
in case of the scenario where just one attribute is added, perhaps we could have the defaults as a fallback?

Reply:
resolved